src/helpers/sortProducts.ts

```
@@ -0,0 +1,40 @@
+  field: keyof ProductListItemVM,
+  direction: 'asc' | 'desc' = 'asc'
+): ProductListItemVM[] {
+  return [...products].sort((a, b) => {
+    const valueA = a[field];
+    const valueB = b[field];
+
+    if (valueA === undefined || valueA === null) return 1;
+    if (valueB === undefined || valueB === null) return -1;
+
+    if (typeof valueA === 'string' && typeof valueB === 'string') {
+      if (!isNaN(Date.parse(valueA)) && !isNaN(Date.parse(valueB))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)